pkg/ctxutil/ctxutil.go

```diff
@@ -153,7 +156,11 @@ func HasGitCommit(ctx context.Context) bool {
 
 // IsGitCommit returns the value of git commit or the default (true).
 func IsGitCommit(ctx context.Context) bool {
-	return is(ctx, ctxKeyGitCommit, true)
+	bv, ok := ctx.Value(ctxKeyGitCommit).(bool)
```

Comment:
The `is` helper should exaclty do this. Why did you replace it?

Reply:
no idea why, but I replaced it back